roles/os_hardening/defaults/main.yml

```diff
@@ -299,27 +296,25 @@ sysctl_config:
 # Do not delete the following line or otherwise the playbook will fail
 # at task 'create a combined sysctl-dict if overwrites are defined'
 sysctl_overwrite:
```

Comment:
should we actually define this as an empty array?

Reply:
I *think* it is an empty string, not array.
Regardless it has always been this way and it works. :)